MIT.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2021, [YOUR NAME]
```

Comment:
- [ ] [OPTIONAL] It would be nice if you add your name instead of`[YOUR NAME}`

Reply:
- This has been resolved.